app/javascript/dashboard/store/modules/conversations/getters.js

```diff
@@ -22,6 +22,21 @@ const getters = {
           CONVERSATION_PRIORITY_ORDER[b.priority]
         );
       },
+      sort_on_waiting_since: (a, b) => {
+        if (!a.waiting_since && !b.waiting_since) {
```

Comment:
```suggestion
        if (!a.waiting_since && !b.waiting_since) {
```

Is there a possibility `waiting_since` can be `0` and can affect this sort method? If not, can we check for null directly?

Reply:
I believe the test case covers the value of 0.

Reply:
@scmmishra It is 0 if the value is nil (since we do waiting_since.to_i everywhere). I have covered this.